modules/registrars/synergywholesaledomains/synergywholesaledomains.php

```diff
@@ -280,6 +280,12 @@ function synergywholesaledomains_getConfigArray(array $params)
             'Type' => 'yesno',
             'Description' => 'Enable this option to force update the WHOIS.json data<br><b>NOTE:</b> This option will be disabled automatically again once you have clicked \'Save Changes\' and the update sequence is completed.',
         ],
+        'defaultDnsConfig' => [
+            'FriendlyName' => 'Default DNS Config',
+            'Type' => 'dropdown',
+            'Options' => ['Nothing', 'Names Servers', 'FreeDNS with email forwarding', 'Parked', 'FreeDNS'],
```

Comment:
We should have the keys for these options be the ID of the corresponding Synergy value, similar to this:

```php
'Options' => [
   '0' => 'None',
   '1' => 'Nameservers'
   '4' => 'FreeDNS'
],

```